ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -152,6 +159,42 @@ public void shouldFailWithEmptyOutput() throws Exception {
 
   }
 
+  @Test
+  public void shouldFailWhenInsertingWrongType() throws Exception {
```

Comment:
might be controversial - but I'd only have one of these three tests and name it `shouldPropegateInsertValuesExecutorError`. Otherwise we're testing functionality that is really implementation details of another class and we're letting that bleed into the tests here.